### DISCUSSION TITLE
GetCourse

### DISCUSSION THREAD

**Discussion by Kosmo-girl** [aclap-dev/video-downloadhelper#1028] [Help and Q&A]
### Your question

I'm interested in how to download a video from a site GetCourse, I was able to do this on another computer, but not now... maybe I need an extension or something else?

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: ru
coapp: "checking"
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 10
}

**Comment by paulrouget** [comment]
See https://github.com/aclap-dev/video-downloadhelper/discussions/972
